share/man/man9/intro.9

```diff
@@ -33,7 +33,7 @@ the first time will be able to understand.
 .Pp
 To further set expectations, we acknowledge that kernel documentation, like the
 source code itself, is forever a work-in-progress.
-There will be large sections of the codebase whose documentation is subtly or
+There will be large sections of the codebase where documentation is subtly or
```

Comment:
I really think "whose" is accurate here, as it conveys the relationship between source code and source code documentation.

Reply:
Both are OK, but I think whose is better and fits with the style better... It is the codebase's documentation that's out of date, fanciful, missing or otherwise a work of fiction.

Reply:
Afterthought: 

> … large sections of the codebase for which documentation is …

Re: <https://english.stackexchange.com/questions/452489/whose-and-of-which/452491#comment1089625_452491>, I tend to avoid _whose_ for inanimate objects.

Reply:
Please drop this change.
https://www.merriam-webster.com/grammar/whose-used-for-inanimate-objects#:~:text=Which%20and%20that%2C%20the%20relative,beginning%20of%20a%20question%3A%20while%20%22
is a better source than stack exchange. There's lots of ways to express this, and the project's other documentation uses whose in this context. It's not wrong and clearer.

Reply:
```suggestion
There will be large sections of the codebase whose documentation is subtly or
```